File: 程式語言 V2答案檔/C#/406.cs
using System;

public class TQC406
{
    public static void Main(string[] args)
    {
        string inputStr;
        string upper = "QWERTYUIOPPASDFGHJKLLZXCVBNMM";
        string lower = "qwertyuioppasdfghjkllzxcvbnmm";
        
        inputStr = Console.ReadLine();
        
        string result = "";
        for (int i = 0; i < inputStr.Length; i++) {
            if (Char.IsLower(inputStr[i])) {
                result += lower[lower.IndexOf(inputStr[i]) + 1];
            }
            else {
                result += upper[upper.IndexOf(inputStr[i]) + 1];
            }
        }

        Console.WriteLine(result);
    }
}

File: 程式語言 V2答案檔/Python/406.py
keyboard = [['q','w','e','r','t','y','u','i','o','p']
            , ['a','s','d','f','g','h','j','k','l']
            , ['z','x','c','v','b','n','m']]

def find_right_alpha(c):
    global keyboard
    output_c = ''
    for i in range(len(keyboard)):
        if c in keyboard[i]:
            idx = keyboard[i].index(c)
            # print(idx)
            if idx == len(keyboard[i]) - 1:
                output_c = keyboard[i][idx]
            else:
                output_c = keyboard[i][idx + 1]
    return output_c

input_str = input()
output_str = ''
for c in input_str:
    if c.isupper():
        c = c.lower()
        # print(c)
        output_str += find_right_alpha(c).upper()
    else:
        output_str += find_right_alpha(c)
print(output_str)
                    
        
